cumulusci/tasks/github/tests/test_merge.py

```diff
@@ -176,14 +184,16 @@ def test_branch_does_not_exist(self):
         task = self._create_task()
         with self.assertRaises(GithubApiNotFoundError):
             task()
-        self.assertEquals(len(responses.calls), 2)
+        self.assertEquals(2, len(responses.calls))
```

Comment:
Just curious… Why switch these around?

Reply:
The output seems to be nicer if you put expected first in the assertEquals call.  Definitely the diff is nicer that way.